.env

```
@@ -17,14 +17,14 @@ NEXT_PUBLIC_DEVELOPMENT_API_URL=http://${WEBSITE_DOMAIN}:${DEV_PORT}
 NEXT_PUBLIC_GRAPHQL_API_URL=http://${WEBSITE_DOMAIN}:${GRAPHQL_PORT}/${GRAPHQL_API_PATH}
 NEXT_PUBLIC_MOCKOON_CONTAINER_API_URL=http://${MOCKOON_HOST}:${MOCKOON_PORT}
 NEXT_PUBLIC_MOCKOON_LOCAL_API_URL=http://${WEBSITE_DOMAIN}:${MOCKOON_PORT}
-NEXT_PUBLIC_PROD_CONTAINER_API_URL=http://${NEXT_PUBLIC_PROD_HOST}:${NEXT_PUBLIC_PROD_PORT}
 NEXT_PUBLIC_PROD_HOST_API_URL=http://${WEBSITE_DOMAIN}:${NEXT_PUBLIC_PROD_PORT}
 
 NEXT_PUBLIC_FALLBACK_LANGUAGE=en
 NEXT_PUBLIC_MAIN_LANGUAGE=uk
 
 NEXT_PUBLIC_CONTINUOUS_DEPLOYMENT_HEADER_NAME=no-aws-header-name
 NEXT_PUBLIC_CONTINUOUS_DEPLOYMENT_HEADER_VALUE=no-aws-header-value
+NEXT_PUBLIC_API_BASE_URL=http://${MOCKOON_HOST}:${MOCKOON_PORT}
```

dotenv-linter (3.3.0)

[warning] [UnorderedKey] The NEXT_PUBLIC_API_BASE_URL key should go before the NEXT_PUBLIC_CONTINUOUS_DEPLOYMENT_HEADER_NAME key

(UnorderedKey)